v19.1/remove-nodes.md

```diff
@@ -69,39 +86,45 @@ To successfully decommission a node, you need to first add a 6th node:
   <button style="width: 15%" class="filter-button" data-scope="insecure">Insecure</button>
 </div>
 
-### Before you begin
+### Before decommissioning a node
 
 To ensure your cluster can adequately handle decommissioning nodes:
 
-- Only decommission one node at a time, and before decommissioning each node verify that there are no [underreplicated or unavailable ranges](admin-ui-replication-dashboard.html).
-- If you have a decommissioning node that appears to be stuck or hung, [contact our support team](support-resources.html).
+- Before decommissioning each node verify that there are no [underreplicated or unavailable ranges](admin-ui-replication-dashboard.html).
+- If you have a decommissioning node that appears to be hung, you can [recommission](#recommission-nodes) the node. If you notice any issues persisting, [contact our support team](support-resources.html).
 
-    If possible, keep the node running instead of killing it, because a stuck decommissioning process might be a symptom of a problem that could result in data loss.
+    If possible, keep the node running instead of stopping it, because a hung decommissioning process might be a symptom of a problem that could result in data loss.
 - Confirm that there are enough nodes to take over the replicas from the node you want to remove. See some [Example scenarios](#examples) above.
 
-### Step 1. Check the node before decommissioning
+### Step 1. Get the ID of the node to decommission
+
+Open the Admin UI and select the **Node List** view, or go to **Metrics** on the left and click **View nodes list** in the **Summary** area. Note the ID of the node that you want to decommission:
+
+<div style="text-align: center;"><img src="{{ 'images/v19.1/before-decommission0.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
+
+### Step 2. Check the node before decommissioning
 
 Open the Admin UI, click **Metrics** on the left, select the **Replication** dashboard, and hover over the **Replicas per Store** and **Leaseholders per Store** graphs:
 
 <div style="text-align: center;"><img src="{{ 'images/v19.1/before-decommission2.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
 
 <div style="text-align: center;"><img src="{{ 'images/v19.1/before-decommission1.png' | relative_url }}" alt="Decommission a single live node" style="border:1px solid #eee;max-width:100%" /></div>
 
-### Step 2. Decommission and remove the node
+### Step 3. Start the decommissioning process on the node
 
-SSH to the machine where the node is running and execute the [`cockroach quit`](stop-a-node.html) command with the `--decommission` flag and other required flags:
+SSH to any live node in the cluster and run the [`cockroach node decommission`](cockroach-node.html) command with the ID of the node to decommission:
```

Comment:
This is a bit weird - we usually do SSH to the node because it's easiest from there to run commands against that node. But if you have the host, and it's reachable from wherever you are, then you can run the command from there instead via the `--host` flag. You specify that below, so it's weird that redundantly you ask users to ssh to a machine. I personally would focus on the `--host` version and avoid recommending to ssh to a machine that runs CRDB since that isn't even always possible (think k8s)